android/build.gradle

```diff
@@ -44,6 +44,6 @@ repositories {
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    compile ('com.facebook.react:react-native:0.55.4') { force = true }
+    compile "com.facebook.react:react-native:+"
```

Comment:
I had to do this the other day because I would get the following error (similar to this picture):
![](https://user-images.githubusercontent.com/5820988/40038682-0a3e1b4c-5831-11e8-85e6-a9cfaad288e4.png)

Reply:
This should not happen in `mobile` anymore. See https://github.com/kiwicom/mobile/blob/master/android/app/build.gradle#L184. 

I suggest remove `node_modules` and `yarn` again. `mobile` should be using in master `0.55.4` so I don't know why you have in JS `0.52.0` ?

Reply:
This screenshot is taken from some other GH thread in some other repo, I just did not have time to take another one, so it is not the actual error I had ;)
I will try to use your commit in the project and see if I get it to work :)